src/main/kotlin/calculator/StringCalculatorProperties.kt

```diff
@@ -0,0 +1,35 @@
+package calculator
+
+private val customDelimiterRegex = "//(.)\\n(.*)".toRegex()
+
+sealed interface StringCalculatorProperties {
+    val delimiters: Array<String>
+    val numberString: String
+
+    fun getPositiveInts(): List<Int> = numberString.split(*delimiters).map { it.toPositiveInt() }
+
+    private fun String.toPositiveInt(): Int = toInt().takeIf { it >= 0 } ?: throw IllegalArgumentException("음수가 아닌 정수를 입력해주세요.")
```

Comment:
저같은 경우에는 각 구문 단위로 [줄 바꿈](https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls)을 해주는것이 가독성이 좋다고 생각하는데 어떻게 생각하시나요??
```suggestion
    private fun String.toPositiveInt(): Int = toInt()
        .takeIf { it >= 0 }
        ?: throw IllegalArgumentException("음수가 아닌 정수를 입력해주세요.")
```

Reply:
[012f0df](https://github.com/next-step/kotlin-lotto/pull/255/commits/012f0df32517818effd1dc6498e32000cbadd3af) 에 반영했습니다!